src/CFixer/main.cpp

```diff
@@ -6,14 +6,46 @@
 //#
 
 #include <quasarapp.h>
+#include <iostream>
+
+void helpCall();
 
 int main(int argc, char *argv[]) {
 
     if (!QuasarAppUtils::Params::parseParams(argc, argv)) {
-        QuasarAppUtils::Params::log("wrong parametrs", QuasarAppUtils::Warning);
-        exit(0);
+        helpCall();
+        return 1;
+    }
 
+    if (QuasarAppUtils::Params::isEndable("h") || QuasarAppUtils::Params::isEndable("help")) {
+        helpCall();
+        return 0;
     }
 
     return 0;
 }
+
+/**
+ * @brief helpCall - a helper call that display infotmation
+ *                   about the arguments and how to use them
```

Comment:
```suggestion
 *  about the arguments and how to use them
```
remove empty spaces from the methods description